docs/src/man/man1/xhc-hb04.1.adoc

```diff
@@ -52,7 +52,7 @@ file.
 
 Usage:
 
-$ xhc-hb04 [options]
+`xhc-hb04` [_options_]
```

Comment:
```suggestion
$ `xhc-hb04` [_options_]
```
I wouldn't remove the "$" as it indicates the command line prompt.

Reply:
The synopsis or usage lines of man pages never show the $ - and back in the days I kept tying a lot with the PS1 variable. That $ should go.